airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/component/TableSchemaEvolutionClient.kt

```diff
@@ -25,7 +25,7 @@ import io.airbyte.cdk.load.table.TableName
  *
  * @see TableOperationsClient for standard SQL-based table operations
  */
-interface TableSchemaEvolutionClient {
+interface TableSchemaEvolutionClient<AdditionalSchemaInfo, AdditionalSchemaInfoDiff> {
```

Comment:
For the 2 example implementations I'm seeing `Unit`, `Unit`. What are some examples of what these might be?

Second nit: `AdditionalSchemaInfo, AdditionalSchemaInfoDiff` are a little verbose for type parameters and I mistook them for concrete types.